CNAME

```diff
@@ -1 +0,0 @@
-www.jjsteyn.co.za
```

Comment:
You need this file. its the thing that tells the website its found on www.jjsteyn.co.za